code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -287,8 +287,91 @@
 	for(var/asym in typesof(/obj/item/clothing/accessory/asymmetric))
 		var/obj/item/clothing/accessory/asymmetric_type = asym
 		asyms[initial(asymmetric_type.name)] = asymmetric_type
-	gear_tweaks += new/datum/gear_tweak/path(sortTim(asyms, /proc/cmp_text_asc, TRUE))
+	gear_tweaks += new/datum/gear_tweak/path(sortTim(asyms, /proc/cmp_text_asc))
 
 /datum/gear/accessory/cowledvest
 	display_name = "cowled vest"
 	path = /obj/item/clothing/accessory/cowledvest
+
+/datum/gear/choker //A colorable choker
+	display_name = "choker (colorable, tagless)"
+	path = /obj/item/clothing/accessory/choker
+	slot = slot_tie
+	sort_category = "Accessories"
+
+/datum/gear/choker/New()
+	..()
+	gear_tweaks = list(gear_tweak_free_color_choice)
+
+/datum/gear/collar
+	display_name = "collar, silver"
+	path = /obj/item/clothing/accessory/collar/silver
+	slot = slot_tie
+	sort_category = "Accessories"
+
+/datum/gear/collar/New()
+	..()
+	gear_tweaks = list(gear_tweak_collar_tag)
+
+/datum/gear/collar/golden
+	display_name = "collar, golden"
+	path = /obj/item/clothing/accessory/collar/gold
+
+/datum/gear/collar/bell
+	display_name = "collar, bell"
+	path = /obj/item/clothing/accessory/collar/bell
+
+/datum/gear/collar/shock
+	display_name = "collar, shock"
+	path = /obj/item/clothing/accessory/collar/shock
+
+/datum/gear/collar/spike
+	display_name = "collar, spike"
+	path = /obj/item/clothing/accessory/collar/spike
+
+/datum/gear/collar/pink
+	display_name = "collar, pink"
+	path = /obj/item/clothing/accessory/collar/pink
+
+/datum/gear/collar/holo
+	display_name = "collar, holo"
+	path = /obj/item/clothing/accessory/collar/holo
+
+/datum/gear/collar/cow
+	display_name = "collar, cowbell"
+	path = /obj/item/clothing/accessory/collar/cowbell
+
+/datum/gear/collar/holo/indigestible
+	display_name = "collar, holo (indigestible)"
+	path = /obj/item/clothing/accessory/collar/holo/indigestible
+
+/datum/gear/accessory/webbing
+	cost = 1
+
+/datum/gear/accessory/stethoscope
+	allowed_roles = list("Chief Medical Officer","Medical Doctor","Chemist","Psychiatrist","Paramedic", "Field Medic")
+
+/datum/gear/accessory/khcrystal
+	display_name = "KH Life Crystal"
+	path = /obj/item/storage/box/khcrystal
+	description = "A small necklace device that will notify an offsite cloning facility should you expire after activating it."
```

Comment:
While you're editing these files can you please scratch out this instance of KHI <3
Don't forget to touch the box's define as well so it doesn't bork
```suggestion
/datum/gear/accessory/vmcrystal
	display_name = "VeyMed Life Crystal"
	path = /obj/item/storage/box/vmcrystal
	description = "A small necklace device that will notify an offsite cloning facility should you expire after activating it."
```

Reply:
Also scratch out the [/datum/gear/accessory/lifecrystal] as it's a bugged copy of this item